src/clj_djl/mmml.clj

```diff
@@ -0,0 +1,73 @@
+(ns clj-djl.mmml
+  (:require [clj-djl.model :as m]
+            [clj-djl.ndarray :as nd]
+            [clj-djl.nn :as nn]
+            [clj-djl.training :as t]
+            [clj-djl.training.dataset :as ds]
+            [scicloj.metamorph.ml :as ml]
+            [tech.v3.dataset :as dataset])
+  (:import java.io.File))
+
+(defn ->ndarray
+  "Convert dataframe to NDArray"
+  [ndm dataframe]
+  (nd/t (nd/create ndm (map vec (dataset/columns dataframe)))))
+
+(defn- train
```

Comment:
My question to you here is:

Is this code "as general" as it can be ?
As you can see here:

https://scicloj.github.io/scicloj.ml/userguide-cljdjl.html


The plugin assumes that this information is passed in:

```clojure
(defn net [] (nn/sequential {:blocks (nn/linear {:units 1})
                         :initializer (nn/normal-initializer)}))
(defn cfg [] (t/training-config {:loss (loss/l2-loss)
                             :optimizer (optimizer/sgd
                                         {:tracker (tracker/fixed learning-rate)})
                             :evaluator (t/accuracy)
                             :listeners (listener/logging)}))

{              :batchsize 64
              :model-spec {:name "mlp" :block-fn net}
              :model-cfg (cfg)
              :initial-shape (nd/shape 1 310)
              :nepoch 1
}
}
```
Does this cover "all networks possible with djl ?

Reply:
I will add more more networks. Currently, it's fine.